package.json

```diff
@@ -10,16 +10,19 @@
   },
   "private": false,
   "scripts": {
-    "build": "webpack --mode production"
+    "build": "webpack --mode development"
   },
   "devDependencies": {
```

Comment:
actually no changes in the devDependencies just put them back in the right order :)